test/webpack.build.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs-extra');
```

Comment:
To the point, the rules you mentioned in the comment above have an effect on code readability / quality for source files and are there for a reason imo.

For something like `webpack.build.js` where it has no effect on the above, I would sooner disable the linting outright in the file than disable rules across the organization that have a positive effect on overall quality.